force-app/main/default/classes/ProgramSelector_TEST.cls

```diff
@@ -69,4 +133,35 @@ public with sharing class ProgramSelector_TEST {
             }
         );
     }
+
+    private static List<Bucket__mdt> createBuckets() {
+        BucketedField__mdt bucketedField = (BucketedField__mdt) new TestUtil.BucketedFieldBuilder()
+            .withDeveloperName('ProgramCohortStatuses')
+            .withQualifiedApiName(Util.prefixNamespace('ProgramCohortStatuses'))
+            .withField('Status__c')
+            .withObject('ProgramCohort__c')
+            .withMockId()
```

Comment:
If we aren't setting a mockId do we need to call this method?

Reply:
Sorry I missed it. I will go ahead and remove it. Thanks for catching it.